pandas/tests/groupby/test_apply.py

```diff
@@ -1151,3 +1151,17 @@ def test_doctest_example2():
         {"B": [1.0, 0.0], "C": [2.0, 0.0]}, index=Index(["a", "b"], name="A")
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("dropna", [True, False])
+def test_apply_dropna_with_indexed_same2(dropna):
```

Comment:
This test completely supersedes the other `test_apply_dropna_with_indexed` test; can you just apply the parametrization and the `expected` change to that test.

Reply:
Amended.
Let me know how this looks.